tools/generator/dfg/stm32/stm_device_tree.py

```diff
@@ -570,6 +593,19 @@ def addDmaToNode(p, driver):
                             rem = sign.addChild("remap")
                             rem.setAttributes(["position", "mask", "id"], remap)
 
+        mux_channels = p.get("dma_mux_channels")
+        if mux_channels is not None:
+            driver_channels = driver.addChild("mux-channels")
+            driver_channels.addSortKey(lambda e: (int(e.get("instance", 0)),
+                                                  int(e.get("position", 0)),
+                                                  int(e.get("dma-instance", 0)),
+                                                  int(e.get("dma-channel", 0))))
+            for channel in mux_channels:
+                chan = driver_channels.addChild("mux-channel")
+                chan.setAttribute("instance", 1)  # H7 has further mux instances for yet unsupported BDMA
```

Comment:
I would only add it for H7, since it's implicit for all other families, and probably a completely different IP?

Reply:
Ok, let's only add it for H7. The MUX is completely identical even for H7. The DMA looks pretty similar as well on a first look. H7 have additional different DMAs, like BDMA for D3 clock domain peripherals and MDMA for accessing TCM memories.